src/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "kaholo-plugin-jira",
+  "version": "1.0.0",
+  "description": "Kaholo Plugin for Jira",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Ilan Yaniv",
+  "license": "ISC",
+  "dependencies": {
+    "jira-client": "^6.17.0",
```

Comment:
@ilanyaniv is `jira-client` necessary? I don't see it used anywhere in the code

Reply:
No, it was for testing.
I will remove it.